.github/workflows/haskell-ci.yml

```diff
@@ -0,0 +1,65 @@
+# This GitHub workflow config has been generated by a script via
```

Comment:
It does not look like this file was generated just by haskell-ci, so this comment is rather outdated and I suggest to remove it.